### PR TITLE
[docs] Migrating Web-UI and Governance Guides to learn

### PR DESCRIPTION
**Pull request by angrycub**
Adding redirect rules and removing legacy guide content

### CODE REVIEW

File: website/source/guides/web-ui/index.html.md [hunk 1]
```diff
@@ -8,12 +8,14 @@ description: |-
...
-1. [Securing the Web UI with ACLs](/guides/web-ui/securing.html)
+You can follow these links to find the specific guides on Learn:
+
+- [Access the Web UI](https://learn.hashicorp.com/nomad/web-ui/access)
+- [Inspect Cluster Workload](https://learn.hashicorp.com/nomad/web-ui/ui-workload) (formerly "Operating a Job")
+- [Inspect the Cluster](https://learn.hashicorp.com/nomad/web-ui/inspecting-the-cluster)
+- [Submit a Job from the Web UI](https://learn.hashicorp.com/nomad/web-ui/submit-job)
+- [Nomad UI Considerations](https://learn.hashicorp.com/nomad/web-ui/considerations)
+
+[HashiCorp's Learn website]: https://learn.hashicorp.com/nomad?track=governance-and-policy#governance-and-policy
```
Comment (DingoEatingFuzz) [inline website/source/guides/web-ui/index.html.md]: This query param looks off.

Comment (angrycub) [reply website/source/guides/web-ui/index.html.md]: It's correct.  It just looks odd.  The track param sets the "active" track when the page renders.  The # jumps to the anchor itself.

> URI = scheme:[//authority]path[?query][#fragment]

Comment (DingoEatingFuzz) [reply website/source/guides/web-ui/index.html.md]: But why is it deep linking to the governance and policy track from the UI page?

Comment (angrycub) [reply website/source/guides/web-ui/index.html.md]: omg.  🤦‍♂   was doing so much work between the 2 tracks I missed what you meant.  fix incoming

**Pull request merged**